lib/board.rb

```diff
@@ -1,9 +1,10 @@
 class Board
   attr_reader :grid
-  attr_accessor :agent
+  attr_accessor :current_player
```

Comment:
This is probably more or less the same thing as you intended, I just changed the language to match how I was thinking about it in the rest of the source. Feel free to adjust to whatever language works for you!

Reply:
I think that's good! I was initially going to use "player" instead of "agent", but thought it could get confusing. Adding "current" to the name makes it just as clear, and less awkward than "agent".